php/lang/security/non-literal-header.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
+- id: non-literal-header
+  patterns:
+  - pattern: header(...)
+  - pattern-not: header("...",...)
+  message: >-
+    Using user input when deleting files with `unlink()` is potentially dangerous. A malicious actor could use this to modify
```

Comment:
I think this message is incorrect?

Reply:
Ooops :sweat_smile: thanks a lot for finding it! updated :+1: